v2/brigadier-polyfill/package.json

```diff
@@ -27,9 +27,10 @@
     "typescript": "^4.1.2"
   },
   "dependencies": {
+    "@brigadecore/brigade-sdk": "^v2.0.0-alpha.5",
+    "@brigadecore/brigadier": "../brigadier",
     "@types/node": "^14.14.11",
     "axios": "^0.21.0",
```

Comment:
I think we might be able to dump axios as a direct dependeny.

Reply:
Good call; removed.

After running `make lint-js`, it looks like the `yarn.lock` file in `v2/worker` was updated as well -- and they look like identical updates to those in `v2/brigadier-polyfill`, I suppose b/c the latter is a dep for the former.  So, I've added these updates as well.